shared/constants/wallets.js

```diff
@@ -595,6 +596,11 @@ const rootWalletPath = [...rootWalletTab, ...(isMobile ? [SettingsConstants.wall
 const isLookingAtWallet = (routeState: ?RouteStateNode) =>
   getPath(routeState, rootWalletTab).get(isMobile ? 2 : 1) === 'wallet'
 
+const loadMobileOnlyWaitingKey = (id: Types.AccountID) => `wallets:getMobileOnly:${id}`
```

Comment:
we don't need these waiting keys as no UI is even hooked up to this. also if we do want this (likely) there should just be done for loading settings